tests/test_classes_second_part.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from codecademy.classes_second_part import Car, ElectricCar, Point3D
+
+test_model = 'bmw'
+test_color = 'Green'
+test_mpg = 100
+test_battery_type = "li"
+
+
+def test_display_car():
+    my_car = Car(test_model, test_color, test_mpg)
+
+    assert my_car.model == test_model
+    assert my_car.color == test_color
+    assert my_car.mpg == test_mpg
+
+    assert my_car.display_car() == "This is a %s %s with %s MPG." % (test_model, test_color, test_mpg)
+
+
+def test_drive_car():
+    test_condition = "used"
+    my_car = Car(test_model, test_color, test_mpg)
+    assert my_car.drive_car() == test_condition
+    assert my_car.condition == test_condition
+
+
+def test_drive_electric_car():
+    test_condition = "like new"
+    my_car = ElectricCar(test_model, test_color, test_mpg, test_battery_type)
+    assert my_car.drive_car() == test_condition
+
+
+@pytest.mark.parametrize("x, y, z,expected", [
+    (0, 0, 0, (0, 0, 0)),
+    (0, 1, 1, (0, 1, 1)),
+    (1, 1, 1, (1, 1, 1)),
+    (-1, -1, 0, (-1, -1, 0)),
+    (123, 222, 999, (123, 222, 999)),
+])
+def test_get_class_average(x, y, z, expected):
+    points = Point3D(x, y, z)
+    assert eval(str(points)) == expected
```

Comment:
Here expected just duplicates your x,y,z.
So just use x,y,z to verify output

Reply:
Done